<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecondWindow</class>
 <widget class="QDialog" name="SecondWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>721</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>712</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(27,29,35,255)</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
font: 9pt &quot;Lato&quot;;
border-radius:30px;
background-color: rgb(34,36,44);
color:#dadada;
padding-left:20px;
padding-right:20px;
}
QTextEdit:Hover {
background-color:rgb(48,50,62);
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fileLocationLine">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>126</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato Light</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <fontweight>Light</fontweight>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	font: 300 9pt &quot;Lato Light&quot;;
border-radius:9.19px;
background-color: rgb(34,36,44);
color:#fff;
padding-left:10px;
padding-right:10px;
}
QLineEdit:Hover{
background-color:rgb(48,50,62);
}</string>
   </property>
   <property name="placeholderText">
    <string>Enter File Location</string>
   </property>
  </widget>
  <widget class="QLabel" name="methodselectionLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>150</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#dadada;
font: 9pt &quot;Lato&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Select Method&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>196</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#dadada;
font: 9pt &quot;Lato&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Enter File Location&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="railFenceCheckBox">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>200</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#fff;
font: 9pt &quot;Lato&quot;;
icon-color:black;</string>
   </property>
   <property name="text">
    <string>Railfence</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browseButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>100</y>
     <width>40</width>
     <height>14</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 9pt &quot;Lato&quot;;
border-radius:9.19px;
background-color: rgb(34,36,44);
color:#FFF
}
QPushButton:Hover{
background-color:rgb(48,50,62);
}


</string>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="caseCheckBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>310</y>
     <width>99</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{
font: 9pt &quot;Lato&quot;;
color:#fff
}
QCheckBox:Hover{
background-color:rbg(98,98,98);

}</string>
   </property>
   <property name="text">
    <string>Preserve Case</string>
   </property>
  </widget>
  <widget class="QLabel" name="depthLabel">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>230</y>
     <width>103</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#dadada;
color: rgb(218, 218, 218);

font: 9pt &quot;Lato&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Input Depth&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="KeyLine">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>126</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato Light</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <fontweight>Light</fontweight>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	font: 300 9pt &quot;Lato Light&quot;;
border-radius:9.19px;
background-color: rgb(34,36,44);
color:#fff;
padding-left:10px;
padding-right:10px;
}
QLineEdit:Hover{
background-color:rgb(48,50,62);
}


</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Enter Key</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="punctuationCheckBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>141</width>
     <height>14</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#fff;
border-radius:60px;
border:none;font: 9pt &quot;Lato&quot;;</string>
   </property>
   <property name="text">
    <string>Preserve Punctutation</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="depthSpinBox">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>270</y>
     <width>38</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox{
	font: 9pt &quot;Lato&quot;;

background-color: rgb(34,36,44);
color:#fff
}
QSpinBox:Hover {
background-color:rgb(48,50,62);
}

</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="vigenereCheckBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>65</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#fff;
icon-color:black;
font: 9pt &quot;Lato&quot;;</string>
   </property>
   <property name="text">
    <string>Vign√®re</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetButtton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>350</y>
     <width>31</width>
     <height>14</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 9pt &quot;Lato&quot;;
border-radius:9.19px;
background-color: rgb(34,36,44);
color:#FFF
}

QPushButton:Hover {
background-color:rgb(48,50,62);
}

</string>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="decryptButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>350</y>
     <width>43</width>
     <height>14</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 9pt &quot;Lato&quot;;
border-radius:9.19px;
background-color: rgb(34,36,44);
color:#FFF
}

QPushButton:Hover {
background-color:rgb(48,50,62);
}
</string>
   </property>
   <property name="text">
    <string>Decrypt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="encryptButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>350</y>
     <width>42</width>
     <height>14</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 9pt &quot;Lato&quot;;

border-radius:9.19px;
background-color: rgb(34,36,44);
color:#FFF

}

QPushButton:Hover {
background-color:rgb(48,50,62);
}

</string>
   </property>
   <property name="text">
    <string>Encrypt</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
